recipe/meta.yaml

```diff
@@ -18,13 +18,14 @@ source:
   sha256: {{ sha256sum }}
 
 build:
-  number: 1
+  number: 2
   skip: true  # [win32]
   skip: true  # [win and py<35]
 
 requirements:
   build:
     - cmake
+    - patchelf
```

Comment:
@cpcloud can you remove this to get a green build again?

Reply:
yep